alcor/services/plots/velocities_vs_magnitude.py

```diff
@@ -152,22 +138,6 @@ def plot_lepine_case(session: Session,
     velocities_w_std = [stars_bin.w_velocity_std
                         for stars_bin in w_vs_mag_bins]
 
-    (u_bins_avg_magnitudes,
-     avg_u_velocities,
-     u_velocities_std) = zip(*sorted(zip(u_bins_avg_magnitudes,
-                                         avg_u_velocities,
-                                         u_velocities_std)))
-    (v_bins_avg_magnitudes,
-     avg_velocities_v,
-     velocities_v_std) = zip(*sorted(zip(v_bins_avg_magnitudes,
-                                         avg_velocities_v,
-                                         velocities_v_std)))
-    (w_bins_avg_magnitudes,
-     avg_velocities_w,
-     velocities_w_std) = zip(*sorted(zip(w_bins_avg_magnitudes,
-                                         avg_velocities_w,
-                                         velocities_w_std)))
-
```

Comment:
так зачем изначально это нужно было?

Reply:
я же написал в комменте к PR:
"We had to sort data when we used Cassandra because it corrupted the order of rows but with Postgres there is no such a problem"